dpsim-villas/examples/cxx/FpgaExample.cpp

```diff
@@ -21,11 +23,17 @@ using namespace CPS::DP::Ph1;
 const std::string buildFpgaConfig(CommandLineArgs &args) {
   std::filesystem::path fpgaIpPath =
       "/usr/local/etc/villas/node/etc/fpga/vc707-xbar-pcie-dino/"
-      "vc707-xbar-pcie-dino.json";
+      "vc707-xbar-pcie-dino-v2.json";
 
   if (args.options.find("ips") != args.options.end()) {
     fpgaIpPath = std::filesystem::path(args.getOptionString("ips"));
   }
+  std::string loopbackConfig = fmt::format(
```

Comment:
DPsim already uses [nlohmann/json](https://github.com/nlohmann/json).

I think it would nicer here to use the JSON library to construct the VILLASnode config.

Reply:
I actually use that library to create json objects. I don't think there is much difference if I use strings or initializer lists. This way makes it more similar to a json file that would be used with VILLASnode.